tests/Unit/Infrastructure/ErrorLoggingFileFetcherTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Unit\Infrastructure;
+
+use FileFetcher\FileFetcher;
+use FileFetcher\FileFetchingException;
+use FileFetcher\InMemoryFileFetcher;
+use Psr\Log\LoggerInterface;
+use WMDE\Fundraising\Frontend\Infrastructure\ErrorLoggingFileFetcher;
+use PHPUnit\Framework\TestCase;
+
+class ErrorLoggingFileFetcherTest extends TestCase {
+
+	public function testGivenSucceedingFileFetcher_itsContentIsReturned() {
+		$logger = $this->createMock( LoggerInterface::class );
+		$logger->method( 'error' )->with( $this->never() );
+		$errorLoggingFileFetcher = new ErrorLoggingFileFetcher(
+			new InMemoryFileFetcher( ['song.txt' => 'I\'m a little teapot'] ),
```

Comment:
Yes, there is a PHPCS sniff that enforces spaces in `[ 'stuff' ]`. It`s MediaWiki.WhiteSpace.SpaceyParenthesis, and requires [mediawiki/mediawiki-codesniffer](https://packagist.org/packages/mediawiki/mediawiki-codesniffer).